test/test_core.py

```diff
@@ -7491,6 +7491,10 @@ def test_embind_negative_constants(self):
     self.emcc_args += ['-lembind']
     self.do_run_in_out_file_test('embind/test_negative_constants.cpp')
 
+  def test_embind_field_access(self):
+    self.emcc_args += ['-lembind']
+    self.do_run_in_out_file_test('embind/test_field_access.cpp', assert_identical=True)
```

Comment:
Most tests don't use `assert_identical=True` .. is there some reason to use it here?